### ISSUE TITLE
Mix up of standalone and format tokens for luxon

### ISSUE THREAD

**Issue by pomadin** [dmtrKovalenko/date-io#31]
Hi, 

It seems that for `@date-iu/luxon` you have setup formats without taking into considearion [the token difference ](https://moment.github.io/luxon/docs/manual/formatting.html#standalone-vs-format-tokens).

So i have found the following formats in sources
```ts
  public yearMonthFormat = "MMMM yyyy";
  public dateFormat = "LLLL dd";
```

I would expect the formats
```ts
  public yearMonthFormat = "LLLL yyyy"; // standalone token for month as it has no correlation with a day
  public dateFormat = "MMMM dd"; // format token for month as it should correlate with day
```

Or am I wrong in my expectations?

**Comment by dmtrKovalenko** [comment]
You are right I suppose... 
We need to change it.

**Comment by pomadin** [comment]
Just to add one more proof
![image](https://user-images.githubusercontent.com/9018346/53552285-90e65900-3b6d-11e9-98ce-06f92bdcc168.png)
It's window calendar with RU locale, there is "LLLL yyyy" and "dd MMMM yyyy" formats in terms of luxon.

**Comment by pomadin** [comment]
Hi, any news on the subj? I could add a pull request on it.

**Comment by dmtrKovalenko** [comment]
Would be cool

**Comment by pomadin** [comment]
OK, I did. Please, let me know if I need to update something.

BTW, I faced an effort to run test on windows platform due to TZ parametrization.

**Comment by pomadin** [comment]
Do you have a plan to release the fix?

**Comment by dmtrKovalenko** [comment]
Released in 1.2.0
